app/src/layouts/cards/components/card.vue

```diff
@@ -124,6 +137,10 @@ function handleClick() {
 		<template v-else>
 			<div v-if="$slots.title" class="title"><slot name="title" /></div>
 			<div v-if="$slots.subtitle" class="subtitle"><slot name="subtitle" /></div>
+			<div v-if="item?.id" class="copy-id-row">
+				<v-icon name="content_copy" small class="copy-id-icon" :aria-label="$t('copy_id')" @click.stop="copyCardId" />
```

Comment:
This button is always present. Maybe we only show it on over? 🤔

Reply:
Depends what that looks like RE layout shift I guess